frontend/src/utils/signin-token/index.js

```diff
@@ -0,0 +1,16 @@
+import * as workerTimers from 'worker-timers'
+
+const checkForTokenInMilliseconds = 2700000
+
+const SignInToken = {
+  refresh: (googleApi, success) => {
+    return workerTimers.setTimeout(() => {
+      googleApi.reloadAuthResponse().then(
+        (response) => success(response.id_token),
+        (err) => console.warn('Token un-refreshable: ', err.message))
```

Comment:
Is there any merit in sending this message to the logger? Or is this an expected thing that might happen often enough it'd be noisy?

Reply:
This is the frontend so the logger is not available.

Reply:
Oh yeah, sorry! 🤦‍♂